src/sentry/utils/redis.py

```diff
@@ -53,27 +54,60 @@ def make_rb_cluster(*args, **kwargs):
     return _make_rb_cluster(*args, **kwargs)
 
 
+class _RBCluster(object):
+    def supports(self, config):
+        return not config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        return _make_rb_cluster(**config)
+
+    def __str__(self):
+        return 'Redis Blaster Cluster'
+
+
+class _RedisCluster(object):
+    def supports(self, config):
+        return config.get('is_redis_cluster', False)
+
+    def factory(self, **config):
+        nodes = map(lambda n: n[1], config.get('hosts').items())
```

Comment:
Isn't this the same as `config.get('hosts').values()`?

I'm not familiar with the options for Redis Cluster, but it seems like we shouldn't try to make the configuration parameters here superficially similar just for consistency (for example, this configuration should probably just include a `nodes` list rather than a `hosts` map that we transform into the node list) unless there was a wider plan for this to simplify the upgrade path or something?